testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -136,7 +136,7 @@ impl LocalSwarmBuilder {
         let root_account = LocalAccount::new(
             diem_sdk::types::account_config::diem_root_address(),
             AccountKey::from_private_key(root_keys.root_key),
-            0,
+            1,
```

Comment:
may I ask the reason we change seq to 1?

Reply:
Seems that after genesis the root account seq number starts at 1 probably because the genesis transaction is from the root account.